guiEditor/src/sharedComponents/popup.ts

```diff
@@ -0,0 +1,80 @@
+export class Popup {

```

Comment:
Should be shared

Reply:
I start to wonder if these files are not there by error. Should the sharedComponents in GUIEditor not part of the PR?

Reply:
I am a bit confused about the way the shared components are being commited. There are shared components (and shared ui components) in both the nodeMaterial and the guiEditor. @sebavan - is it right to assume that the underlining link will resolve those to be the same directory eventually? 

If this is the case - can we decide on a single place in the repository where those files are commited? Maybe outside of both the node editor and the gui editor?

Reply:
There is a sharedUIComponents folder outside of NME, the inspector, and GUI. It is part completed. Ones that are already been moved I can try to link those up.